frontend/packages/operator-lifecycle-manager/src/components/package-manifest.tsx

```diff
@@ -2,216 +2,144 @@ import * as React from 'react';
+  PackageManifestKind,
+  { catalogSource: CatalogSourceKind }
+> = ({ obj: packageManifest, index, key, style, customData }) => {
+  const channel = !_.isEmpty(packageManifest.status.defaultChannel)
+    ? packageManifest.status.channels.find(
+        (ch) => ch.name === packageManifest.status.defaultChannel,
+      )
+    : packageManifest.status.channels[0];
 
-  const createSubscriptionLink = () =>
-    `/k8s/ns/${ns === ALL_NAMESPACES_KEY ? defaultNS : ns}/${SubscriptionModel.plural}/~new?pkg=${
-      obj.metadata.name
-    }&catalog=${catalogSourceName}&catalogNamespace=${catalogSourceNamespace}`;
+  const { displayName, version, provider } = channel.currentCSVDesc;
 
   return (
-    <TableRow id={obj.metadata.uid} index={index} trKey={rowKey} style={style}>
+    <TableRow id={packageManifest.metadata.uid} index={index} trKey={key} style={style}>
       <TableData className={tableColumnClasses[0]}>
         <ClusterServiceVersionLogo
           displayName={displayName}
-          icon={_.get(icon, '[0]')}
+          icon={iconFor(packageManifest)}
           provider={provider.name}
         />
       </TableData>
       <TableData className={tableColumnClasses[1]}>
         {version} ({channel.name})
       </TableData>
       <TableData className={tableColumnClasses[2]}>
-        {subscription ? subscriptionLink() : <span className="text-muted">None</span>}
-        {canSubscribe && (
-          <Link to={createSubscriptionLink()}>
-            <Button variant="primary" type="button">
-              Create<span className="visible-lg-inline"> Subscription</span>
-            </Button>
-          </Link>
-        )}
+        <Timestamp timestamp={packageManifest.metadata.creationTimestamp} />
       </TableData>
+      {!customData.catalogSource ? (
+        <TableData className={tableColumnClasses[3]}>
+          <ResourceLink
+            kind={referenceForModel(CatalogSourceModel)}
+            name={packageManifest.status?.catalogSource}
+            namespace={packageManifest.status?.catalogSourceNamespace}
+          />
+        </TableData>
+      ) : null}
     </TableRow>
   );
 };
 
-export const PackageManifestList = requireOperatorGroup((props: PackageManifestListProps) => {
-  type CatalogSourceInfo = {
-    displayName: string;
-    name: string;
-    publisher: string;
-    namespace: string;
-  };
-  const catalogs = (props.data || []).reduce(
-    (allCatalogs, { status }) =>
-      allCatalogs.set(status.catalogSource, {
-        displayName: status.catalogSourceDisplayName,
-        name: status.catalogSource,
-        publisher: status.catalogSourcePublisher,
-        namespace: status.catalogSourceNamespace,
-      }),
-    new Map<string, CatalogSourceInfo>(),
-  );
-
+export const PackageManifestList = (props: PackageManifestListProps) => {
+  const { customData } = props;
+  const TableHeader = customData.catalogSource
+    ? PackageManifestTableHeader
+    : PackageManifestTableHeaderWithCatalogSource;
```

Comment:
This is expression is a bit confusing given the names of the props and components. I'd add a comment here to clarify.

Reply:
Added comment:  `If the CatalogSource is not present, display PackageManifests along with their CatalogSources (used in PackageManifest Search page)`